validator/src/lib.rs

```diff
@@ -73,17 +73,17 @@ mod validation;
 pub use validation::cards::validate_credit_card;
 pub use validation::contains::validate_contains;
 pub use validation::does_not_contain::validate_does_not_contain;
-pub use validation::email::validate_email;
+pub use validation::email::{validate_email, ValidateEmail};
 pub use validation::ip::{validate_ip, validate_ip_v4, validate_ip_v6};
-pub use validation::length::validate_length;
+pub use validation::length::{validate_length, ValidateLength};
 pub use validation::must_match::validate_must_match;
 #[cfg(feature = "unic")]
 pub use validation::non_control_character::validate_non_control_character;
 #[cfg(feature = "phone")]
 pub use validation::phone::validate_phone;
```

Comment:
I'm tempted to remove that one from validator, it's currently too generic to be easily usable and can easily be added manually if needed.

Reply:
Maybe leaving an example on how to add it with a custom validator after removing it would be a good idea?